data/json/items/comestibles/med.json

```diff
@@ -1210,7 +1210,6 @@
         "id" : "gummy_vitamins",
         "copy-from": "vitamins",
         "type" : "COMESTIBLE",
-        "comestible_type" : "MED",
```

Comment:
That's the beauty of strict parsing - it doesn't allow you to add spurious redundancy